server/src/main/java/org/elasticsearch/cluster/metadata/ShutdownShardMigrationStatus.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.cluster.metadata;
```

Comment:
I think we'll want to move these out of `o.e.cluster.metadata` since they're not being used for metadata anymore. Maybe `o.e.xpack.shutdown`, since I don't think we need to access them from `server` anymore.

Reply:
Okay, I'm not sure yet where they should move, but we can probably move them out whenever we actually make use of them?

Reply:
SGTM as long as we remember to move them :)